hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/annotations/CustomOps.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2025 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.contract.impl.annotations;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import com.hedera.node.app.service.contract.impl.exec.operations.CustomCallOperation;
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import javax.inject.Qualifier;
+
+/**
+ * Qualifies a set of {@link org.hyperledger.besu.evm.operation.Operation}s to use to further customize
+ * EVM behavior, beyond the standard Hedera customizations like {@link CustomCallOperation}.
+ */
+@Target({METHOD, PARAMETER, TYPE})
+@Retention(RUNTIME)
+@Documented
+@Qualifier
+public @interface CustomOps {}
```

Comment:
_apropos_ of nothing: `InitialTokenServiceApi` annotation in this package is not used, do we still need it?

Reply:
True, we can remove this.